frontend/src/pages/WhoWeAre/whoWeAre.jsx

```diff
@@ -1,13 +1,38 @@
 import './whoWeAre.css';
+import Image from '../../components/Image/image';
 
 /**
- * WhoWeAre component
+ * WhoWeAre page
+ * "Хто ми" (Who We Are) page that explains the platform's mission, origin, and goals,
+ * So that user can understand the purpose and background of
+ * CraftMerge and gain trust in the initiative.
+ *
+ * @component
  * @returns {JSX.Element}
- * @constructor
  */
 function WhoWeAre() {
     return (
-        <></>
+        <div className={'who-we-are'}>
+            <div className={'who-we-are__container'}>
+                <h2 className={'who-we-are__title'}>Хто ми</h2>
+                <div className={'who-we-are__content'}>
+                    <div className={'who-we-are__text'}>
+                        <p>
+                            <span className={'who-we-are__text--bold'}>CraftMerge</span> - перший форум Західної України, який створений у співпраці з Національним університетом «Львівська політехніка». Наша місія - не лише об'єднання українських виробників та стартапів, а й відкриття нових перспектив у виробничій галузі.
+                        </p>
+                        <p>
+                            CraftMerge - це не лише платформа для обміну досвідом та ідеями, але й комунікаційний майданчик для обговорення актуальних тенденцій та передових технологій.
+                        </p>
+                        <p>
+                            Учасники форуму отримають можливість не лише обмінятися досвідом та ідеями, але й ознайомитися з найсучаснішими рішеннями виробництва крафтової продукції. Ми створили CraftMerge, щоб допомогти українським виробникам збільшити свою популярність та впізнаваність, розширити аудиторію споживачів та залучити нових клієнтів. Приєднуйтеся до нашого форуму та розвивайте свій бізнес разом з нами!"
+                        </p>
```

Comment:
For improved readability and accessibility, consider breaking up long paragraphs into shorter ones and using semantic HTML elements such as <strong> or <em> where applicable, instead of only relying on CSS classes for bold text.